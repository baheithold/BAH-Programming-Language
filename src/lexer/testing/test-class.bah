# This file tests valid class


class Rectangle {
    var width;
    var height;
    #...
}
