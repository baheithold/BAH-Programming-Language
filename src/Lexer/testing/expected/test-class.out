CLASS_KEYWORD
ID_TYPE Rectangle
OBRACE
VAR_KEYWORD
ID_TYPE width
SEMICOLON
VAR_KEYWORD
ID_TYPE height
SEMICOLON
CBRACE
END_OF_INPUT
